pkg/config/servicebuild.go

```diff
@@ -108,129 +109,259 @@ func (bldr *ServiceBuilder) WithDataService() *ServiceBuilder {
 	return bldr
 }
 
+func (bldr *ServiceBuilder) WithDB() *ServiceBuilder {
+	bldr.handlers = append(bldr.handlers, func() error {
+		awsSession, err := bldr.Session()
+		if err != nil {
+			return err
+		}
+
+		// TODO: Load these env vars from the ServiceBuilder
```

Comment:
I'm a little confused about when I can and can't use dependencies within a service definition.
I'd rather do `WithEnv` here, but I don't think it would work.
@nathanagood maybe you can correct me on that.